openmdao/devtools/make_hohmann_figures.py

```diff
@@ -0,0 +1,293 @@
+import os.path
```

Comment:
in this case it seems to me that the plotting code belongs with the example code in examples... no?  My thinking would be that it's application-specific and not generally useful for openmdao development...